federation-js/src/composition/__tests__/composeAndValidate.test.ts

```diff
@@ -524,6 +559,16 @@ describe('composition of value types', () => {
       expect(compositionResult.errors[0]).toMatchInlineSnapshot(`
         Object {
           "code": "VALUE_TYPE_FIELD_TYPE_MISMATCH",
+          "locations": Array [
+            Object {
+              "column": 1,
+              "line": 6,
+            },
+            Object {
+              "column": 1,
+              "line": 6,
+            },
+          ],
```

Comment:
Should these locations point to the fields in question instead? (Or better yet the fields' return types)